### ISSUE TITLE
osism apply operator not working with --ask-pass

### ISSUE THREAD

**Issue by berendt** [osism/issues#101]
```
osism apply operator -l 'all:!manager' -u install --ask-pass --ask-become-pass
/usr/lib/python3.8/getpass.py:91: GetPassWarning: Can not control echo on the terminal.
```

**Comment by berendt** [comment]
Workaround: ```osism-generic operator -l 'all:!manager' -u install --ask-pass --ask-become-pass```

**Comment by berendt** [comment]
Finally solved with 5.2.0. We now have a vault command in the CLI.
